graphdatascience/algo/algo_proc_runner.py

```diff
@@ -3,11 +3,11 @@
 
 from pandas import DataFrame, Series
 
+from ..call_parameters import CallParameters
 from ..error.illegal_attr_checker import IllegalAttrChecker
 from ..graph.graph_object import Graph
 from ..graph.graph_type_check import graph_type_check
 from ..model.graphsage_model import GraphSageModel
-from graphdatascience.call_parameters import CallParameters
```

Comment:
realized, we had a mixture of relative and absolute imports for these. 
As the majority was relative, I aligned them